ktor-network/jvm/src/io/ktor/network/sockets/Builders.kt

```diff
@@ -79,8 +80,10 @@ class TcpSocketBuilder internal constructor(
     suspend fun connect(
         hostname: String,
         port: Int,
+        socketTimeout: Long = 0L,
+        parentContext: CoroutineContext = EmptyCoroutineContext,
         configure: SocketOptions.TCPClientSocketOptions.() -> Unit = {}
```

Comment:
Nice to have `socketTimeout` in `SocketOptions`

Reply:
Done.